tests/test_cli.py

```diff
@@ -46,32 +48,117 @@ def assert_expected_file(self, result, output_path):
         except:
             pass
 
-    def test_schema_convert_cli(self, runner, helpers):
-        data_model_csv_path = helpers.get_data_path("example.model.csv")
+    @pytest.mark.parametrize(
+        "output_path",
+        [
+            "tests/data/example.model.pickle",
+            "tests/data/example.model.jsonld"
+        ],
+        ids=["output_path_pickle", "output_path_jsonld"]
+    )
+    @pytest.mark.parametrize(
+        "output_type",
+        [
+            "jsonld",
+            "graph",
+            "all"
+        ],
+        ids=["output_type_jsonld", "output_type_graph", "output_type_all"]
+    )
+    def test_schema_convert_cli(self, runner, output_path, output_type):
```

Comment:
Nit: These are different test cases, per set of parameters.  I'm not certain it's required to split each parameter setting into separate tests, but it would make it more readable.